homework/index.js

```diff
@@ -30,15 +32,116 @@
     return elem;
   }
 
+  function makeHeader(repos) {
+    const root = document.getElementById('root');
+    const header = createAndAppend('header', root, { class: 'header' });
+    createAndAppend('h3', header, { text: 'HYF Repositories' });
+    const selectMenu = createAndAppend('select', header, { class: 'repo-selector' });
+    const optionsArr = [];
```

Comment:
You can get rid of the complete `optionArr` argument: use the `repos` array directly:

```js
function makeHeader(repos) {
  const root = document.getElementById('root');
  const header = createAndAppend('header', root, { class: 'header' });
  createAndAppend('h3', header, { text: 'HYF Repositories' });
  const selectMenu = createAndAppend('select', header, { class: 'repo-selector' });
  repos.map((repo, index) =>
    createAndAppend('option', selectMenu, {
      value: index,
      text: repo.name,
    }),
  );
  return header;
}
```

Reply:
I solved it but I forgot comment here